guide/additional-info/reactions.md

```diff
@@ -253,10 +253,10 @@ We are checking for `emoji.id` because a unicode emoji won't have an ID inside t
 All that's left is to fetch the actual reaction from the message and emit the event.
 
 ```js
-const reaction = message.reactions.get(emoji);
+const reaction = message.reactions.get(emojiKey);
 ```
 
-<p class="tip">In the master branch/v12 reactions are keyed by their ID only, not by it's name:ID.</p>
+<p class="tip">In the master branch/v12, reactions are keyed by their ID only, not in a `name:ID format.</p>
```

Comment:
Missing the closing backtick here, causing it to [not render as markdown](https://i.imgur.com/n8USgkC.png).
(last one, I promise lol)

Reply:
It's fine haha